service/.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I approved and merged the code and then I saw this file. I think this file is committed to github unintentionally. If so, please remove this file.

Reply:
This has been removed and a .vscode setting set in gitignore